app/views/shared/_stats.html.erb

```diff
@@ -17,29 +17,7 @@
 
       <div class="mostFocus">
         <% most_focus.each do |key, value| %>
-          <div class="someFocus">
-            <div class="someFocusText">
-              <% if local_assigns[:data_type] == 'category' && (!local_assigns[:profile] || local_assigns[:profile] == current_user.id) %>
-                <%= viewers_hover(get_viewers_for(Category.where(id: key).first, 'category'), Category.where(id: key).first) %>
-              <% elsif local_assigns[:data_type] == 'category' && local_assigns[:profile] %>
-                <div class="someFocusTextName">
-                  <%= Category.where(id: key).first.name %>
-                </div>
-              <% elsif local_assigns[:data_type] == 'mood' && (!local_assigns[:profile] || local_assigns[:profile] == current_user.id) %>
-                <%= viewers_hover(get_viewers_for(Mood.where(id: key).first, 'mood'), Mood.where(id: key).first) %>
-              <% elsif local_assigns[:data_type] == 'mood' && local_assigns[:profile] %>
-                <div class="someFocusTextName">
-                  <%= Mood.where(id: key).first.name %>
-                </div>
-              <% elsif local_assigns[:data_type] == 'strategy' && (!local_assigns[:profile] || local_assigns[:profile] == current_user.id) %>
-                <%= viewers_hover(get_viewers_for(Strategy.where(id: key).first, 'strategy'), Strategy.where(id: key).first) %>
-              <% elsif local_assigns[:data_type] == 'strategy' && local_assigns[:profile] %>
-                <div class="someFocusTextName">
-                  <%= Strategy.where(id: key).first.name %>
-                </div>
-              <% end %>
-            </div>
-          </div>
+          <%= render 'shared/focus' %>
```

Comment:
Does the page render properly out of curiosity?
In our codebase, we've been doing `<%= render partial: '/shared/stats', locals: { data_type: 'category' } %>` for example.

Reply:
You should really be running the app locally on your machine while making changes to it. Unfortunately, we don't have automated tests for everything. Things like UI often times have to be tested manually.

Are you using Windows or Linux? I recommend using Linux and following our [Installation](https://github.com/ifmeorg/ifme/wiki/Installation) steps for that.

Reply:
I'm using a windows, still working to get a dual boot (Really need to keep using windows for now), Once i get it i'd be sure to install ruby on rails

Reply:
Sounds good! I also recommend using Linux in VirtualBox. It takes a lot less time than doing a dual boot installation.

Setting up Ruby on Rails on Windows is possible but not as straightforward compared to Linux or macOS.